packages/material-ui/src/Divider/Divider.js

```diff
@@ -20,14 +22,28 @@ export const styles = theme => ({
     left: 0,
     width: '100%',
   },
-  /* Styles applied to the root element if `inset={true}`. */
+  /* Styles applied to the root element if `variant={'inset'}`. */
   inset: {
     marginLeft: 72,
```

Comment:
We should perhaps have an example of overriding the width:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/357702/48444757-c0bf4b80-e78c-11e8-80d0-1c7740e15317.png">